examples/query_filter.R

```diff
@@ -0,0 +1,20 @@
+library(magrittr)
+library(cohortBuilder)
+pkgload::load_all()
```

Comment:
```suggestion
library(shinyCohortBuilder)
```

Use library in the examples.

Reply:
done